src/graphs/Breadth_First_Search.rei

```diff
@@ -0,0 +1,9 @@
+type node = {
+    id: string,
+    neighbours: list(string),
+};
+type returnResult = {
+    level: Hashtbl.t(string, int), 
+    parent: Hashtbl.t(string, string)
+};
+let breadthFirstSearch: (list(node), string) => returnResult;
```

Comment:
You should change the module name to `BreadthFirstSearch`.
Maybe change this function to `search`?